wallet/src/de/schildbach/wallet/ui/TransactionResultActivity.kt

```diff
@@ -52,11 +52,11 @@ class TransactionResultActivity : AbstractWalletActivity() {
 
         @JvmStatic
         fun createIntent(context: Context, transaction: Transaction, userAuthorized: Boolean,
-                         payeeName: String? = null, payeeVerifiedBy: String? = null): Intent {
+                         paymentMemo: String? = null, payeeVerifiedBy: String? = null): Intent {
             val transactionResultIntent = Intent(context, TransactionResultActivity::class.java)
             transactionResultIntent.putExtra(EXTRA_TX_ID, transaction.txId)
             transactionResultIntent.putExtra(EXTRA_USER_AUTHORIZED_RESULT_EXTRA, userAuthorized)
-            transactionResultIntent.putExtra(EXTRA_PAYEE_NAME, payeeName)
+            transactionResultIntent.putExtra(EXTRA_PAYMENT_MEMO, paymentMemo)
```

Comment:
This section will have merge conflicts with #406 .

Reply:
I merged `deeplink-payment-fix` into `bip70-updates` and resolved conflicts.
This PR should be merged before #406.

Reply:
If `deeplink-payment-fix` was merged into this PR, shouldn't #406 be merged to master before this PR #407?  Otherwise it seems that #406 would be an empty merge since all the code was merged with #406.  In any case, I will try it out locally before doing it on GitHub.